pages.ru/common/tput.md

```diff
@@ -0,0 +1,28 @@
+# tput
+
+> Просмотр и изменение настроек и возможностей терминала.
+> Больше информации: <https://manned.org/tput>.
+
+- Переместить курсор в определённое место на экране:
+
+`tput cup {{номер_строки}} {{номер_столбца}}`
+
+- Установить цвет переднего плана (af) или фона (ab):
+
+`tput {{setaf|setab}} {{ansi_код_цвета}}`
+
+- Показать количество столбцов, строк или цветов:
+
+`tput {{cols|lines|colors}}`
+
+- Подать звуковой сигнал терминала:
+
+`tput bel`
+
+- Сбросить все атрибуты терминала:
+
+`tput sgr0`
+
+- Включить/отключить перенос слов:
+
+`tput {{smam|rmam}}`
```

Comment:
```suggestion
- Включить или отключить перенос слов:

`tput {{smam|rmam}}`
```

Optional change (after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/stty.md) page fix): `или` can be used instead of forward slash.

Reply:
The link above shoud be https://github.com/tldr-pages/tldr/blob/main/pages/common/tput.md

Reply:
Thanks for mentioning my mistake. Fixed